src/components/media-player/ha-media-player-browse.ts

```diff
@@ -259,11 +269,10 @@ export class HaMediaPlayerBrowse extends LitElement {
                         <div class="ha-card-parent">
                           <ha-card
                             outlined
-                            style=${styleMap({
-                              backgroundImage: child.thumbnail
-                                ? `url(${child.thumbnail})`
-                                : "none",
-                            })}
+                            class=${child.thumbnail ? "lazythumbnail" : ""}
+                            data-src=${child.thumbnail
+                              ? `${child.thumbnail}`
```

Comment:
No need to wrap in string interpolation
```suggestion
                              ? child.thumbnail
```